index.html

```diff
@@ -122,21 +122,21 @@ <h2>Introduction</h2>
 
       <p>
 Reading and navigating technical specifications can be confusing. For this
-reason we have created this guide, which provides some examples and resources
+reason, we have created this guide, which provides some examples and resources
 for implementing protocols which make use of <a>verifiable credentials</a>.
       </p>
       <p>
-It may be useful to become familiarized with the official
+It may be useful to firt familiarize yourself with the official
```

Comment:
`firt` -> `first`

Reply:
hah, oops. Thanks. Fixed in [e796e9a](https://github.com/w3c/vc-imp-guide/pull/26/commits/e796e9a5b3d3cca46face05b8a6e925687ea6d4d)